### PR TITLE
Hafiz user setting form

### PR DESCRIPTION
**Pull request by hafijul233**
this push has user show page layout change
- added a setting menu
- password reset and  locale change function added

### CODE REVIEW

File: resources/lang/bd/auth.php [hunk 1]
```diff
@@ -26,6 +26,10 @@
         'success' => 'প্রমাণপ্রত্রাদি যাচাই করা হয়েছে. স্বাগত !',
         'banned' => 'ব্যবহারকারী অ্যাকাউন্ট নিষ্ক্রিয় করা হয়েছে। সিস্টেম প্রশাসকের সাথে যোগাযোগ করুন।',
         'forced' => 'আপনার পাসওয়ার্ড পরিবর্তন করা হয়েছে। পাসওয়ার্ড রিসেট করুন.'
+    ],
+    'logout' => [
+        'title' => 'ত্যাগের জন্য প্রস্তুত?',
+        'message' => 'আপনি যদি আপনার বর্তমান অধিবেশন শেষ করতে প্রস্তুত হন তবে নীচে "সাইন আউট" নির্বাচন করুন৷'
```
Comment (mah-shamim) [inline resources/lang/bd/auth.php:32]: ত্যাগের -> সাইন আউট

File: app/Http/Requests/Backend/Setting/UserSettingRequest.php [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+            $rules['username'][] = 'required';
+        else :
+            $rules['username'][] = 'nullable';
+        endif;
+
+        return $rules;
+    }
+
+
+    protected function prepareForValidation()
```
Comment (mah-shamim) [inline app/Http/Requests/Backend/Setting/UserSettingRequest.php:68]: need documentation

**Review by mah-shamim — CHANGES_REQUESTED** [review]
please flow those review

File: resources/lang/bd/auth.php (hunk 1 above)
Comment (hafijul233) [reply resources/lang/bd/auth.php:32]: ok i will change those translations

**Review by mah-shamim — APPROVED** [review]
done

**Comment by hafijul233** [comment]
thank u for your approval. please merge it

**Pull request merged**